<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainW_ui1</class>
 <widget class="QMainWindow" name="MainW_ui1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainW_ui1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="checkBox1">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>85</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton1">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>98</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>431</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>98</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest_Dialog">
    <property name="title">
     <string>Test Dialog</string>
    </property>
    <addaction name="actionTest1"/>
   </widget>
   <addaction name="menuTest_Dialog"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTest1">
   <property name="text">
    <string>Test1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
